Curtains/Curtains.Infrastructure/Repositories/AccessoriesRepository.cs

```diff
@@ -39,6 +32,12 @@ public AccessoriesRepository(CurtainsDbContext curtainsContext, ILogger logger)
         /// <returns>Collection of AccessoriesModel entities in List type</return>
         public IEnumerable<AccessoriesModel> GetAll()
         {
+            if (!_curtainsContext.Accessories.Any())
+            {
+                _logger.LogError("Accessories table is empty");
+                throw new ResourceNotFoundException();
+            }
+
```

Comment:
For what reason you throw exception, if we can return empty list

Reply:
in other methods we throw exception